src/client/unity.yml

```diff
@@ -6,6 +6,31 @@ servers:
   - url: /unity/v2 # to be determined
 paths:
   "/orgs/{orgID}/invites":
+    get:
+      operationId: GetInvites
+      tags:
+        - Invites
+      parameters:
+        - $ref: "#/components/parameters/TraceSpan"
+        - in: path
+          name: orgID
+          description: Specifies the organization ID of the Invite.
+          required: true
+          schema:
+            type: string
+      responses:
+        "200":
+          description: A list of cloud invites
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Invites"
+        default:
+          description: Unexpected error
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/Error"
```

Comment:
So separate endpoint for pulling invites rather than getting both users and invites with the same endpoint? That makes a lot of sense! 👍